networkit/cpp/graph/test/GraphGTest.cpp

```diff
@@ -40,7 +40,8 @@ class GraphGTest : public testing::TestWithParam<std::tuple<bool, bool>> {
 
     bool isWeighted() const;
     bool isDirected() const;
-    Graph createGraph(count n = 0) const;
+    Graph createGraph(node n = 0) const;
+    Graph createGraph(node n, count m) const;
```

Comment:
About the types of `n` and `m`: if we are using `node` for `n`, shouldn't we use `edgeid` for `m`? A simpler choice could be `count` for both.